portfolio/src/main/webapp/script.js

```diff
@@ -135,20 +143,58 @@ function createMarkers(myMap){
       marker: new google.maps.Marker({position: {lat: 38.9072, lng: -77.0369 }, map: myMap}),
       description: "DC is about 20 minutes from my house. Main attractions here are the White House, Lincoln Memorial, 9-11 Memorial, and the US Monument."
     },
-  ]
+  ];
 
   for (let index=0; index<myMarkers.length; index++){
     createInfoWindows(myMap, myMarkers[index].marker, myMarkers[index].description);
   }
 }
 /**
  * Adds Info Windows for markers.
+ * @param {map} map - Our google map.
+ * @param {marker} marker - The marker we want to attach an info window to.
+ * @param {string} description - The description of the marker.
  */
 function createInfoWindows(map, marker, description){
   const infoWindow = new google.maps.InfoWindow({content: description});
   marker.addListener("click", () => {
       infoWindow.open(map, marker);
   });
 }
+/**
+ * Creates chart and adds it to page.
+ */
+function drawChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Island');
+  data.addColumn('number', 'count');
```

Comment:
What is this a count of?

Reply:
the average enjoyment of people who went to the islands :)